docs/.doxygen/Doxyfile

```diff
@@ -32,19 +32,19 @@ DOXYFILE_ENCODING      = UTF-8
 # title of most generated pages and in a few other places.
 # The default value is: My Project.
 
-PROJECT_NAME           = "HIP: Heterogenous-computing Interface for Portability"
+PROJECT_NAME           = "HIP Runtime API Reference"
 
 # The PROJECT_NUMBER tag can be used to enter a project or revision number. This
 # could be handy for archiving the generated documentation or if some version
 # control system is used.
 
-PROJECT_NUMBER         =
+PROJECT_NUMBER         = v3.0.1.0
```

Comment:
Why is there a version number here?

Reply:
Removed now. I reviewed the changes to the doxyfile to only change relevant values.